src/lib/components/Filteration.svelte

```
@@ -68,13 +68,15 @@
-				url.searchParams.delete(itemName + '_ids', String(children.id));
+				if (filteration) {
+					const url = new URL(page.url);
+					url.searchParams.delete(itemName + '_ids', String(children.id));
 
-				// reset pagination to first page when filters removed
-				url.searchParams.set('page', '1');
+					// reset pagination to first page when filters removed
+					url.searchParams.set('page', '1');
 
-				goto(url, { keepFocus: true, invalidateAll: true });
+					goto(url, { keepFocus: true, invalidateAll: true });
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)